src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,44 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
+    private static final int DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int HOURS_INDEX = 2;
+    private static final int SALARY_INDEX = 3;
+    private static final String INFO_SEPARATOR = " ";
+    private static final String EXPECTED_SEPARATOR = " - ";
+
+    private final DateParser dateParser = new DateParser();
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate startDate = dateParser.date(dateFrom);
+        LocalDate stopDate = dateParser.date(dateTo);
+        int [] salaries = new int[names.length];
```

Comment:
```suggestion
        int[] salaries = new int[names.length];
```
However, we could easily avoid using this array

Reply:
Hmm, avoiding is absolutely not obvious to me, I'll try to figure it out, thanks!

Reply:
I could introduce local variable, right!